failures when building C++

While running `make cpp/output.dso, the build process dies with this error:

```
Error: In file included from ./src/snikket/jingle/_PeerConnection/CppByte_Impl_.cpp:9:
include/snikket/jingle/_PeerConnection/CppByte_Impl_.h:58:50: error: 'std::byte' has not been declared
   58 |                 static unsigned char toInt( std::byte this1);
      |                                                  ^~~~
./src/snikket/jingle/_PeerConnection/CppByte_Impl_.cpp:34:15: error: 'unsigned char snikket::jingle::_PeerConnection::CppByte_Impl__obj::toInt' is not a static data member of 'class snikket::jingle::_PeerConnection::CppByte_Impl__obj'
   34 | unsigned char CppByte_Impl__obj::toInt( std::byte this1){
      |               ^~~~~~~~~~~~~~~~~
./src/snikket/jingle/_PeerConnection/CppByte_Impl_.cpp:34:46: error: 'byte' is not a member of 'std'
   34 | unsigned char CppByte_Impl__obj::toInt( std::byte this1){
      |                                              ^~~~
./src/snikket/jingle/_PeerConnection/CppByte_Impl_.cpp:34:46: note: 'std::byte' is only available from C++17 onwards
```

--- a/README.md
+++ b/README.md
@@ -4,14 +4,21 @@ https://github.com/snikket-im/snikket-sdk
 
 Working towards simplicity in developing Snikket-compatible apps.
 
-    haxelib git jsImport https://github.com/back2dos/jsImport
-    haxelib install datetime
-    haxelib install haxe-strings
-    haxelib install hsluv
-    haxelib install tink_http
-    haxelib install sha
-    haxelib install thenshim
-    haxelib install HtmlParser
+  haxelib setup ~/haxe
+  haxelib --quiet git jsImport https://github.com/back2dos/jsImport
+  haxelib --quiet install datetime
+  haxelib --quiet install haxe-strings
+  haxelib --quiet install hsluv
+  haxelib --quiet install tink_http
+  haxelib --quiet install sha
+  haxelib --quiet install thenshim
+  haxelib --quiet install HtmlParser
+  haxelib --quiet install hxnodejs
+  haxelib --quiet git hxtsdgen https://github.com/singpolyma/hxtsdgen
+  haxelib --quiet install utest
+  haxelib --quiet git hxcpp https://github.com/HaxeFoundation/hxcpp
+  cd ~/haxe/hxcpp/git/tools/hxcpp
+    haxe compile.hxml
     make
 
 # JavaScript

